### PR TITLE
Changed fragment_upload_list.xml from Relative to Constraint layout

### PR DESCRIPTION
**Pull request by Anniekobia**
**What this PR does**
Changed fragment_upload_list.xml from Relative layout to Constraint layout

**Reference**
[Sub issue] #148 

**Screenshots**
User interface remains the same

### CODE REVIEW

File: app/src/main/res/layout/fragment_upload_list.xml [hunk 1]
```diff
@@ -8,23 +9,38 @@
...
 
     </androidx.recyclerview.widget.RecyclerView>
 
     <TextView
         android:id="@+id/tv_no_uploads"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
         android:layout_centerInParent="true"
+        android:layout_marginEnd="411dp"
+        android:layout_marginBottom="731dp"
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/fragment_upload_list.xml]: This won't look good for all screen sizes. Please make this Textview at the center.

Comment (Anniekobia) [reply app/src/main/res/layout/fragment_upload_list.xml]: Thanks for the feedback. I have corrected that and pushed the changes.

**Comment by ujjwalagrawal17** [comment]
Thanks @Anniekobia ! Congrats on your first PR to the project.

**Comment by Anniekobia** [comment]
Thank you @ujjwalagrawal17

**Pull request merged**